saga/seata-saga-statemachine-designer/src/utils/index.js

```diff
@@ -60,12 +60,12 @@ export function setProperties(businessObject, properties, override) {
 export function is(element, target) {
   const type = element?.businessObject?.Type || element?.Type || element;
 
-  if (target === 'State') {
-    return type === 'ServiceTask';
+  if (target === 'Task') {
```

Comment:
this modification is expect?

Reply:
Yes. This function is to check the type for an element. For now, I need to know if an element is a ‘Task’ or not, not if it is a 'State'.